README.md

```diff
@@ -69,84 +69,82 @@ cat /.../src/dump.sql | docker exec -i nerdery-container psql -U postgres -d ner
 
 ## 📊 Excersises <a name = "excersises"></a>
 
-Now it's your turn to write SQL querys to achieve the following results:
+Now it's your turn to write SQL queries to achieve the following results (You need to write the query in the section `Your query here` on each question):
 
-1. Count the total number of states in each country.
+1. Total number of accounts.
 
 ```
 Your query here
 ```
 
-<p align="center">
- <img src="src/results/result1.png" alt="result_1"/>
-</p>
 
-2. How many employees do not have supervisores.
+2. How many users with at least one 2 `CURRENT_ACCOUNT`.
 
 ```
 Your query here
 ```
 
-<p align="center">
- <img src="src/results/result2.png" alt="result_2"/>
-</p>
 
-3. List the top five offices address with the most amount of employees, order the result by country and display a column with a counter.
+3. List the top five accounts with more amount.
 
 ```
 Your query here
 ```
 
-<p align="center">
- <img src="src/results/result3.png" alt="result_3"/>
-</p>
 
-4. Three supervisors with the most amount of employees they are in charge.
+4. Get the three users with the most value after making movements.
 
 ```
 Your query here
 ```
 
-<p align="center">
- <img src="src/results/result4.png" alt="result_4"/>
-</p>
 
-5. How many offices are in the state of Colorado (United States).
+5. In this part you need to create a transaction with the following steps:
 
-```
-Your query here
-```
+    a. First, get the ammount for the account `3b79e403-c788-495a-a8ca-86ad7643afaf` after all their movements.
+    b. Add a new movement with the information:
+        type: IN
+        mount: 50.75
 
-<p align="center">
- <img src="src/results/result5.png" alt="result_5"/>
-</p>
+    c. Add a new movement with the information:
+        type: OUT
+        mount: 731823.56
+
+        * Note: if the account does not have enough money you need to reject this insert and make a rollback for the entire transaction
+    
+    d. Put your answer here if the transaction fails(YES/NO):
+    ```
+        Your answer
+    ```
+
+    e. If the transaction fails, make the correction on step _c_ to avoid the failure:
+    ```
+        Your query
+    ```
 
-6. The name of the office with its number of employees ordered in a desc.
+    f. Once the transaction is correct, make a commit
+    ```
+        Your query
+    ```
+
+
+6. All the movements and the user information with the account `3b79e403-c788-495a-a8ca-86ad7643afaf`
 
 ```
 Your query here
 ```
 
-<p align="center">
- <img src="src/results/result6.png" alt="result_6"/>
-</p>
 
-7. The office with more and less employees.
+7. The name and email of the user with the highest amount on his/her all accounts
```

Comment:
The name and email of the user with the highest amount in all his/her accounts

Reply:
Changed